src/StartingEnv.hs

```diff
@@ -360,6 +360,7 @@ dynamicModule = Env { envBindings = bindings
                     , makePrim "eval" 1 "evaluates a list." "(eval mycode)" primitiveEval
                     , makePrim "defined?" 1 "checks whether a symbol is defined." "(defined? mysymbol)" primitiveDefined
                     , makePrim "deftemplate" 4 "defines a new C template." "(deftemplate symbol Type declString defString)" primitiveDeftemplate
+                    , makePrim "implements" 2 "designates a function as an implementation of an interface" "(implements zero Maybe.zero)" primitiveImplements
```

Comment:
Nitpick: there should be a period at the end of the documentation sentence.

Reply:
Done, nice catch, thanks!